packages/altair-electron/src/app/touchbar.ts

```diff
@@ -20,9 +20,16 @@ export class TouchbarManager {
 
     const showDocsButton = new TouchBarButton({
       label: 'Show Docs',
+      enabled: !this.windowInstance.webContents.showDocs,
       click: () => this.actionManager.showDocs(),
     });
 
+    const hideDocsButton = new TouchBarButton({
+      label: 'Hide Docs',
+      enabled: this.windowInstance.webContents.showDocs,
```

Comment:
Did you test this? I don't think `this.windowInstance.webContents.showDocs` exists